image/cli/mascli/functions/must_gather

```diff
@@ -93,6 +108,13 @@ function mustgather() {
 
   TIMESTAMP=$(date +"%Y%m%d-%H%M%S")
 
+  # use the provided login details to authenticate against OCP
+  export OCP_LOGIN_URL=$OCP_LOGIN_URL
+  export OCP_LOGIN_TOKEN=$OCP_LOGIN_TOKEN
+  export OCP_DISABLE_TLS_VERIFY=$OCP_DISABLE_TLS_VERIFY
+
+  connect
```

Comment:
If we are running mustgather in the cluster then it will already have a connection, but the call to `connect` here will mean that it will prompted the user to confirm. Can we change this so we only connect if we have the OCP_LOGIN_URL and OCP_LOGIN_TOKEN set?